content/en/about/feature-stages/index.md

```diff
@@ -36,7 +36,8 @@ Below is our list of existing features and their current phases. This informatio
 | Protocols: Websockets / MongoDB  | Stable
 | Traffic Control: label/content based routing, traffic shifting | Stable
 | Resilience features: timeouts, retries, connection pools, outlier detection | Stable
-| Gateway: Ingress, Egress for all protocols | Stable
+| Gateway Functionality (Ingress, Egress for all protocols) | Stable
+| Gateway APIs (Ingress, Egress for all protocols) | Beta
```

Comment:
Maybe i am alone in thinking this, as its not written down anywhere, but my impression was `apiVersion` != feature status phase. The networking APIs are stable whether we call it that in the apiVersion (which is due to a technical constraint of the CRD API) or not.

For example, thinking we could deprecate the Gateway API in 3 months is completely impractical.

Reply:
Thanks. So we currently have: 

```
apiVersion: networking.istio.io/v1beta1
kind: VirtualService
```

But the API itself is stable.